src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1023,27 +1022,23 @@ public static bool IsDefaultExpressionContext(this SyntaxTree syntaxTree, int po
                 token.Parent.IsParentKind(SyntaxKind.Parameter) &&
                 token.Parent.GetParent().GetParent().IsDelegateOrConstructorOrLocalFunctionOrMethodOrOperatorParameterList(includeOperators))
```

Comment:
some really wacky mixing of Parent and GetParent :)

Reply:
`GetParent` is null-resilient

Reply:
Actually you're right... `token.Parent.Parent` can't be null (because of the `IsParentKind` above) and since it's not null we can get its `Parent` as well.

Reply:
right :)

Reply:
➡️ Fixed in c19fca102a667180ae9f5ba2a113127a15474f0d